packages/vitest/src/index.ts

```diff
@@ -24,73 +24,87 @@ declare module 'vite' {
   }
 }
 
+interface AsymmetricMatchersContaining {
+  stringContaining(expected: string): void
+  objectContaining(expected: any): ObjectContaining
+  arrayContaining(expected: unknown[]): ArrayContaining
+  stringMatching(expected: string | RegExp): StringMatching
+}
+
 declare global {
   namespace Chai {
-    interface ExpectStatic {
+    interface ExpectStatic extends AsymmetricMatchersContaining {
       extend(expects: MatchersObject): void
-      stringContaining(expected: string): void
+      assertions(expected: number): void
+      hasAssertions(): void
       anything(): Anything
-      objectContaining(expected: any): ObjectContaining
       any(constructor: unknown): Any
-      arrayContaining(expected: any): ArrayContaining
-      stringMatching(expected: RegExp): StringMatching
-      assertions(expected: number): void
+      not: AsymmetricMatchersContaining
     }
 
-    interface Assertion {
-      // Chai
-      chaiEqual(expected: any): void
-
+    interface JestAssertions<T = void> {
```

Comment:
Why is T needed?

Reply:
I used it before writing `Promisify` helper and was lazy to removed 👀